migrations/0032-add-created-at-column-to-merch-collection.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddCreatedAtColumnToMerchCollection1640851101278 implements MigrationInterface {
+  name = 'AddCreatedAtColumnToMerchCollection1640851101278';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query('ALTER TABLE "MerchandiseCollections" ADD "createdAt" TIMESTAMP NOT NULL DEFAULT now()');
+  }
+
```

Comment:
For dates in our database we use the `timestamptz` instead of the `timestamp` column type, so let's use that type as well for `createdAt` columns.